database/seeders/FacultySeeder.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Faculty;
+use Illuminate\Database\Seeder;
+
+class FacultySeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $faculties = ['Faculty of Humanities','Faculty of Social and Management Science','Faculty of Science and Science Education'];
```

Comment:
You don't need to put "Faculty of ". Just call it "Humanities", "Social and Management Sciences", "Science and Science Education"